.babelrc

```diff
@@ -1,7 +1,3 @@
 {
-  "presets": [["env", {
-    "targets": {
-      "uglify": true
-    }
-  }]]
+  "presets": [["@babel/preset-env"]]
```

Comment:
I suggest changing this to:

```
{
  "presets": [["@babel/env", {
    "forceAllTransforms": true
  }]]
}

This will make sure babel generates ES5 (and thus UglifyJS) compatible code.